src/main/java/pl/zz/sharefood/service/FoodUpdateService.java

```diff
@@ -0,0 +1,29 @@
+package pl.zz.sharefood.service;
+
+import java.util.Date;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import pl.zz.sharefood.domain.Food;
+import pl.zz.sharefood.repository.FoodRepository;
+
+@Service
+@RequiredArgsConstructor
+public class FoodUpdateService {
+
+  private final FoodRepository foodRepository;
+
+  public Food updateFood(Food newFood, Long id) {
```

Comment:
Jeżeli nie znajdziesz rekordu to rzuć wyjątek i zwróć komunikat do response. Poza tym jest coś takiego jak saveOrUpdate, wtedy spróbuje zaktualizować rób dodać zasób